migration_guides/70.x.x-to-71.0.0-migration-guide.md

```diff
@@ -0,0 +1,389 @@
+                {{
+                    onsHeader({
+                        "title": 'Page title',
+                        "mastheadLogoUrl": {
+                            "multipleLogos": {
+                                "logo1": {
+                                    "logoURL" : "https..."
+                                }
+                            }
+                        },
+                        "titleUrl": '#0',
+                        "language": {
+                            "languages": [
+                                {
+                                    "url": '#0',
+                                    "ISOCode": 'en',
+                                    "text": 'English',
+                                    "current": true
+                                },
+                                {
+                                    "url": '#0',
+                                    "ISOCode": 'cy',
+                                    "text": 'Cymraeg',
+                                    "current": false
+                                }
+                            ]
+                        },
+                        "navigation": {
+                            "subNavigation": {
+                                "overviewURL": "https..."
+                            }
+                        }
+                    })
+                }}
+                NEW
+                {{
+                    onsHeader({
+                        "title": 'Page title',
+                        "mastheadLogoUrl": {
+                            "multipleLogos": {
+                                "logo1": {
+                                    "logoUrl" : "https..."
+                                }
+                            }
+                        },
+                        "titleUrl": '#0',
+                        "language": {
+                            "languages": [
+                                {
+                                    "url": '#0',
+                                    "isoCode": 'en',
+                                    "text": 'English',
+                                    "current": true
+                                },
+                                {
+                                    "url": '#0',
+                                    "isoCode": 'cy',
+                                    "text": 'Cymraeg',
+                                    "current": false
+                                }
+                            ]
+                        },
+                        "navigation": {
+                            "subNavigation": {
+                                "overviewUrl": "https..."
+                            }
+                        }
+                    })
+                }}
+                ```
+
+        </details>
+
+## Components
+
+This section will list all the changes that are specific to one component.
+
+### **_onsCallToAction_**
+
```

Comment:
think we should add in the short title as you have in 'global changes' - as there could be multiple changes per component